tests/console/firewalld.pm

```diff
@@ -63,11 +63,7 @@ sub check_rules {
     } else {
         assert_script_run("nft list chain inet firewalld filter_IN_public_allow | grep 25");
         assert_script_run("nft list chain inet firewalld filter_IN_public_allow | grep 110");
-        if (is_leap("<16.0") || is_sle("<16")) {
```

Comment:
Did you do verification runs with this on leap 15.x / SLE15SPx?

Reply:
Yes, with Leap 15.5